lib/multiset.rb

```diff
@@ -17,6 +17,10 @@ def initialize(*args, &block) #:nodoc:
     super
   end
 
+  def include?(e)
+    @hash[e] != 0
```

Comment:
Actually, I'm going to change this to `@hash.include?(e)` since that's exactly what it was doing prior to ruby 2.3.